content/applications/hr/payroll/payslips.rst

```diff
@@ -0,0 +1,402 @@
+correct month, and click on the corresponding day for both the start and end dates.
+
+.. image:: payslips/new-batch-details.png
+   :align: center
+   :alt: Enter the details for the new batch.
+
+If the batch is a credit note, check the box next to :guilabel:`Credit Note`. Then, in the
+:guilabel:`Generation Date` field, select the date the payslips should be generated from a calendar
+pop-up window. This generated date is reflected in the accounting journal entries.
+
+Lastly, in the :guilabel:`Company` field, select the company these payslips are written against.
+
+When all the information on the batch payslip detail form is correct, click the :guilabel:`Save`
+button to save the information. To delete the entry, click the :guilabel:`Discard` button.
+
+To create the payslips for the newly-created batch, click the :guilabel:`Generate Payslips` button
+at the top of the form.
+
+When clicked, a pop-up window appears showing all the payslips that will be created. To remove any
+individual payslips, click the black :guilabel:`🗙 (clear)` icon at the far right of the payslip
+line.
+
+If a specific :guilabel:`Salary Structure` or :guilabel:`Department` needs to be specified for the
+batch, select them from the corresponding drop-down menus.
+
+Click the :guilabel:`Generate` button at the bottom of the pop-up window to create the payslips for
+the batch.
+
+.. image:: payslips/generate-payslips.png
+   :align: center
+   :alt: Generata payslips for the new batch.
+
+.. note::
+   On the :guilabel:`Generate Payslips` pop-up window, selecting a :guilabel:`Department` and/or
+   :guilabel:`Salary Structure` only displays payslips that apply to those specifically-selected
+   parameters.
+
+If there are any errors or issues preventing the payslips from being generated, a detailed error
+message appears in the top-right section. This error box disappears on its own after several
+seconds, or the 'X' can be clicked to close the alert.
+
+To remedy the issue, make any necessary modifications (e.g. removing any payslip lines that cannot
+be processed), then click the :guilabel:`Generate` button again.
+
+Once the payslips have been successfully generated, the screen returns to the batch payslip detail
+form.
+
+From here, click the :guilabel:`Generate Draft Entry` button to change the payslips status from
+*Draft* to *Done*.
+
+Once payslips have been generated, click the :guilabel:`Make Payment` button to process the
+payments. Doing so reveals a pop-up window, in which the proper banking information must be entered.
+In this pop-up window, select the appropriate :guilabel:`Bank Journal` from the drop-down menu, and
+enter the appropriate file name.
+
+When done, click the :guilabel:`Confirm` button to confirm the information, or click
+:guilabel:`Cancel` to discard.
+
+.. image:: payslips/confirm-payslips.png
+   :align: center
+   :alt: Confirm the bank details for the new payslips.
```

Comment:
I also feel like this image can be removed since your instruction is sufficient :) 

```suggestion
```